projects.html

```diff
@@ -61,8 +62,11 @@
             <div class="electron"></div>
             <div class="electron-alpha"></div>
             <div class="electron-omega"></div>
-        </div>
-    </div>
+		</div>
+		<div>
+			<p id="animate" style="opacity: 0;" class="text-anime">I could either watch it happen or be a part of it.</p>
```

Comment:
This does not make sense at all and invalidates the entire point of having an animation!

Please remove the line

Reply:
This line adds the quote you wanted in the fadein animation. Which line do you want removed?

Reply:
And the id and class tags are later used for the script to run. How will the animation occur then?

Reply:
Having the quote beside the animation makes the UI look worse. Put it either above or below the atom animation

Reply:
Aahh gotcha. Yeah I'll do that